src/core/resource.cpp

```diff
@@ -174,6 +174,8 @@ EMGMResponse CResource::changeFBExecutionState(EMGMCommandType paCommand){
             conn->readData(*getDI(i));
           }
         }
+      }else if(EMGMCommandType::Reset == paCommand){
```

Comment:
@azoitl correct me if I am wrong, but I thought stopped we have using Yoda conditions?

Reply:
Yes we stopped that. @MandKastner please change to `(paCommand == EMGCommandType::Reset)`